docs/operator-manual/airgap/02-install.md

```diff
@@ -125,15 +125,23 @@ helm install --wait -n kubewarden \
 helm install --wait -n kubewarden \
   kubewarden-defaults kubewarden-defaults.tgz \
   --set common.cattle.systemDefaultRegistry=<REGISTRY.YOURDOMAIN.COM:PORT>
```

Comment:
This is needed now, otherwise the command will fail:

```suggestion
  --set common.cattle.systemDefaultRegistry=<REGISTRY.YOURDOMAIN.COM:PORT> \
```